.babelrc

```diff
@@ -0,0 +1,30 @@
+{
+  "presets": [
+    [
+      "env",
+      {
+        "targets": {
+          "browsers": [
+            "last 2 versions",
+            "ie 11"
+          ]
+        }
+      }
+    ],
+    "babel-preset-react"
+  ],
+  "plugins": [
+    "transform-object-rest-spread",
+    "transform-class-properties"
+  ],
+  "env": {
+    "i18n": {
+      "plugins": [
+        ["react-intl", {
+          "messagesDir": "./i18n/default/",
+          "enforceDescriptions": true
+        }]
+      ]
+    }
+  }
```

Comment:
I'm not super familiar with babel, so there's a better-than-zero chance that this `.babelrc` could be written more cleanly/correctly.